src/lib/build.ts

```diff
@@ -112,6 +125,8 @@ export const runBuild = async (options) => {
       scheme: apiUrl.protocol.slice(0, -1),
       host: apiUrl.host,
     }
+    // @ts-expect-error(serhalp) -- I don't know what's going on here and I can't convince myself it even works as
+    // intended. TODO(serhalp) Investigate and fix types.
```

Comment:
Sorry, I had to cut my losses here eventually. It's sad but I'm still trading 13 for 1...